api/src/server/config.rs

```diff
@@ -0,0 +1,117 @@
+//! API config.
+
+use std::net::SocketAddr;
+use std::time::Duration;
+
+use color_eyre::Result;
+use figment::providers::{Env, Serialized};
+use figment::Figment;
+use serde::{Deserialize, Serialize};
+
+/// Runtime configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+pub struct Config {
+    /// Bind address for API server.
+    pub bind: SocketAddr,
+    /// Duration the session(token) is valid.
+    #[serde(with = "humantime_serde")]
+    pub token_timeout: Duration,
+    /// MongoDB connection string.
+    pub mongo_uri: String,
+    /// MongoDB database name.
+    pub mongo_db: String,
+    /// Secret password used to authenticate API requests from bot. This is also used to sign JWT tokens.
+    pub bot_password: String,
```

Comment:
> I only found this to be used as secret of jwt. What about renaming this field to `jwt_secret` and update the comment?

Speak of which, I'm thinking of using asymmetric encrypt for JWT instead of password, do you think that's neccessary?

Reply:
umm nope?